api/proto/config/cipherinfo.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright(c) 2020 Zededa, Inc.
+// All rights reserved.
+
+
+syntax = "proto3";
+
+option go_package = "github.com/lf-edge/eve/api/go/config";
+option java_package = "com.zededa.cloud.uservice.proto";
+//common cipher info for all
+//sensitive data.
+
+enum KeyExchangeAlgoTypes {
+  KEA_NONE  = 0;
+  KEA_ECDH  = 1;
+}
+
+enum SymmetricAlgoTypes {
+  SA_NONE  = 0;
+  SA_AES_256_CFB = 1;
+}
+
+message CipherInfo {
+  KeyExchangeAlgoTypes KeyExchangeAlgo = 1;
+  SymmetricAlgoTypes SymmetricAlgo = 2;
+  bytes initialValue = 3;
+  bytes publicKey = 4;
```

Comment:
What public key is this? The sending zedcloud server?

Do we need to have a way to verify that the result is valid so we don't end up decrypting and producing random bytes?